CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - For the contract checking `new_setattr` function, any variables that depend only on `klass` are now defined in the
   outer function, efficiency of code was improved, and the attribute value is now restored to the original value if the
   `_check_invariants` call raises an error.
+- Can now check explicitly for whether the representation invariants of an object are satisfied.
```

Comment:
You should name the function explicitly: "Added `check_invariants` function that...